### DISCUSSION TITLE
Error on reject or approve transaction

### DISCUSSION THREAD

**Discussion by Sriharsh1103** [WalletConnect/walletconnect-monorepo#2998] [Developer Support] [answered]
![error walletconnect](https://github.com/WalletConnect/walletconnect-monorepo/assets/133873781/8309955a-b909-4c75-af6c-e58ea2370fb4)

whenever we are rejecting or approving the transaction from metamask it is throwing this error

No matching key. history: 1688541113259917

**Comment by SilentRhetoric** [comment]
I get a similar error upon approving a transaction from Defly

```index.ts:5 Uncaught (in promise) Error: No matching key. history: 1691367264856342
    at Lt2.getRecord (index.ts:5:16)
    at Lt2.get (index.ts:5:16)
    at as2.onRelayEventResponse (index.ts:5:16)
    at EventEmitter.<anonymous> (index.ts:5:16)
```

**Comment by glitch-txs** [comment, marked as answer]
Aswered [here](https://github.com/orgs/WalletConnect/discussions/3291). Seems to be an issue on the Wallet side.
